Modelica/Magnetic/FluxTubes/Examples/BasicExamples/QuadraticCoreAirgap.mo

```diff
@@ -0,0 +1,153 @@
+within Modelica.Magnetic.FluxTubes.Examples.BasicExamples;
+model QuadraticCoreAirgap "Educational example: iron core with airgap"
+  extends Icons.Example;
+  parameter Modelica.SIunits.Length l=0.1 "Outer length of iron core";
+  parameter Modelica.SIunits.Length a=0.01 "Quadratic cross section";
+  parameter Real my_r=1000 "Relative permeability of core";
```

Comment:
Could we use Modelica.SIunits.RelativePermeability for this

Reply:
good point. Should be used all over FluxTubes.

Reply:
OK could you update all the PRs regarding this?

Reply:
I'd prefer to do that not on single PRs but after all PRs are merged.